pkgs/by-name/op/openmesh/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  cmake,
+  ...
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
You can wrap with `finalAttrs:` instead of using `rec` and then access version via `finalAttrs.version`.